harcownik-frontend/src/components/userRequests/UserRequests.css

```diff
@@ -1,14 +1,12 @@
 .ApplicationStatusStyle {
```

Comment:
I prefer have more css like names of classes. in this example it will be .application-status-style

Reply:
but now we can stay with this version. In future will be good to change it.